modules/transcription-service-amberscript/src/main/java/org/opencastproject/transcription/amberscript/AmberscriptTranscriptionService.java

```diff
@@ -234,6 +236,16 @@ public void activate(ComponentContext cc) {
     }
     logger.info("Cleanup result files after {} days.", cleanupResultDays);
 
+    Option<String> numberOfSpeakersOpt = OsgiUtil.getOptCfg(cc.getProperties(), NUMBER_OF_SPEAKERS_CONFIG);
+    if (numberOfSpeakersOpt.isSome()) {
+      try {
+        numberOfSpeakers = Integer.parseInt(numberOfSpeakersOpt.get());
+      } catch (NumberFormatException e) {
+        logger.warn("Configured '{}' is invalid. Using default.", NUMBER_OF_SPEAKERS_CONFIG);
```

Comment:
If the configuration is actually really broken, I think the service should refuse to accept it and fail and not fall back to a probably unexpected value.

Though my suggestion is to not have this setting anyway :D
See above

Reply:
In my opinion, fail is too harsh for a problem like this, I agree with @marwyg to let it as default. From what I understand, it is a metadata value.